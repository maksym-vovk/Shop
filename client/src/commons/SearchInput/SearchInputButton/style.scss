.search-btn {
  display: flex;
  align-items: center;
  background: none;
  border: none;
  outline: none;
}