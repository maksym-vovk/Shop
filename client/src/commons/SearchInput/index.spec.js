import React from 'react';
import SearchInputButton from './SearchInputButton';

describe('SearchInputButton component', () => {
  it('SearchInputButton component render', () => {
    const wrapper = shallow(<SearchInputButton />);
    expect(wrapper).toMatchSnapshot();
  });
});