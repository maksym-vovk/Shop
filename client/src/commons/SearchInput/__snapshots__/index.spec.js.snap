// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchInputButton component SearchInputButton component render 1`] = `ShallowWrapper {}`;
